### DISCUSSION TITLE
Option to permanently hide "press for menu" button

### DISCUSSION THREAD

**Discussion by cl8n** [ppy/osu#28049] [Ideas] [answered]
I never use this button. it just keeps distracting me when I'm playing a map with notes near the bottom-right corner, and i see the button fade in and out quickly as my cursor goes near it

**Comment by cl8n** [comment]
bump bc this used to just be a visual annoyance, but since I've posted this I also lost a few plays to unfortunately placed objects that partially overlapped the button (4:3 window) and caused me to actually press it...

**Reply by peppy** [reply]
Hmm, starting to think we can probably hide it by default when touch input is not detected. I'll look at adding a setting.

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/pull/29508
